File: WEEK 1/C++/Hard Drive Storage Capacity Conversion.cpp
#include <iostream>
using namespace std;

int main() {
    double manufacturerSizeGB;

    cout << "Enter the hard drive size in GB (manufacturer's specification): ";
    cin >> manufacturerSizeGB;

    double actualSizeGB = manufacturerSizeGB * 1024;

    printf("Actual storage capacity: %.2f GB\n", actualSizeGB);

    return 0;
}

File: WEEK 1/C#/Hard Drive Storage Capacity Conversion.cs
using System;

class Program {
    static void Main() {
        Console.Write("Enter the hard drive size in GB (manufacturer's specification): ");
        double manufacturerSizeGB = double.Parse(Console.ReadLine());

        double actualSizeGB = manufacturerSizeGB * 1024;

        Console.WriteLine($"Actual storage capacity: {actualSizeGB} GB");
    }
}